### PR TITLE
Fix a memory leak and update release notes for v0.19.0

### PR DESCRIPTION
**Pull request by gigony**
* Fix memory leak in out-of-boundary case
* Do not depend on CuPy for root module
* Update release notes for v0.19.0

### CODE REVIEW

File: python/cucim/src/cucim/__init__.py [hunk 1]
```diff
@@ -13,46 +13,24 @@
...
 
 Most functions are not provided via the top level-import. Instead, individual
 subpackages should be imported instead.
 
 Subpackages
 -----------
-numpy
-    Functions from NumPy which are not available via CuPy.
-scipy
-    Functions from SciPy which are not available via CuPy.
```
Comment (grlee77) [inline python/cucim/src/cucim/__init__.py:29]: thanks for finding and removing these!

File: python/cucim/README.md [hunk 2]
```diff
@@ -2,57 +2,67 @@
...
 
 ```
 pip install cucim
+
+# Install dependencies for `cucim.skimage` (assuming that CUDA 11.0 is used for CuPy)
+pip install scipy scikit-image cupy-cuda110==9.0.0b3
 ```
+### Jupyter Notebooks
+
+Please check out [Welcome](https://github.com/rapidsai/cucim/blob/branch-0.20/notebooks/Welcome.ipynb) notebook.
```
Comment (grlee77) [inline python/cucim/README.md:24]:
```suggestion
Please check out our [Welcome](https://github.com/rapidsai/cucim/blob/branch-0.20/notebooks/Welcome.ipynb) notebook.
```

**Review by grlee77 — APPROVED** [review]
Thanks @gigony. This looks good to me.

**Comment by quasiben** [comment]
I think we should merge this in to 0.20 then create a separate PR for 0.19.  @rapidsai/ops do we still have time to get a hotfix in for 0.19 ?

**Review by quasiben — APPROVED** [review]

File: python/cucim/README.md (hunk 2 above)
Comment (gigony) [reply python/cucim/README.md]: Updated. Thank you!

**Comment by raydouglass** [comment]
> do we still have time to get a hotfix in for 0.19 ?

Sure, ideally get it in by the end of the week

**Comment by quasiben** [comment]
Thanks @raydouglass .  @gigony after this goes in to 0.20 we can make a separate PR with this and the doc fixes as well

**Comment by gigony** [comment]
> after this goes in to 0.20 we can make a separate PR with this and the doc fixes as well

Thanks @raydouglass !
@quasiben Sure. Let me create a PR with this and other doc fixes once this is merged. 
Thanks guys!

@jakirkham Looks like this needs your help (approval) to get merged.

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Gigon! 😄

**Comment by quasiben** [comment]
I think we need someone from @rapidsai/ops to merge in

**Review by raydouglass — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks all! 😄

**Comment by jakirkham** [comment]
Here's the hot fix PR ( https://github.com/rapidsai/cucim/pull/11 )

**Pull request merged**